projects/public-search/src/app/document-brief/document-brief.component.html

```diff
@@ -16,62 +16,55 @@
+      <!-- contribution -->
+      <ul class="list-inline mb-0" *ngIf="record.metadata.contribution && record.metadata.contribution.length > 0">
+        <li
+            class="list-inline-item"
+            *ngFor="let contribution of record.metadata.contribution.slice(0,3); let last = last; let i = index"
+          >
+          <span *ngIf="!contribution.agent.pid; else contribLink">
+            {{ contribution | contributionFormat }}
+          </span>
+          <ng-template #contribLink>
+          <a
+            *ngIf="contribution.agent.pid"
+            href="/{{ viewcode }}/{{ contribution.agent.type | contributionType }}/{{ contribution.agent.pid }}"
+          >
+            {{ contribution | contributionFormat }}
+          </a>
+          </ng-template>
+          {{ last ? '' : '; ' }}
+          {{ last && record.metadata.contribution.length > 3 ? '; …' : '' }}
+        </li>
+      </ul>
 
-        <!-- provision activity publications -->
-        <ul class="list-unstyled" *ngIf="provisionActivityPublications.length > 0">
-          <li *ngFor="let publication of provisionActivityPublications">{{ publication.value }}</li>
-        </ul>
+      <!-- provision activity publications -->
+      <ul class="list-unstyled" *ngIf="provisionActivityPublications.length > 0">
+        <li *ngFor="let publication of provisionActivityPublications">{{ publication.value }}</li>
+      </ul>
 
-        <div *ngIf="record.metadata.type !== 'ebook'">
-          <i class="fa fa-circle text-{{ record.metadata.available ? 'success' : 'danger' }}" aria-hidden="true"></i>&nbsp;
-          <span translate *ngIf="record.metadata.available">available</span>
-          <span translate *ngIf="!record.metadata.available">not available</span>
-        </div>
-        <div *ngIf="record.explanation">
-          <a class="badge badge-info collapsed"
-              data-toggle="collapse" href="#{{'score'+record.metadata.pid }}"
-              aria-expanded="false">
-              score: {{ record.explanation.value }}
-          </a>
-          <pre class="collapse border border-secondary mt-1" id="{{'score'+record.metadata.pid }}">{{record.explanation|json}}</pre>
-        </div>
-      </article>
-    </div>
+      <div *ngIf="record.metadata.type !== 'ebook'">
+        <i class="fa fa-circle text-{{ record.metadata.available ? 'success' : 'danger' }}" aria-hidden="true"></i>&nbsp;
+        <span translate *ngIf="record.metadata.available">available</span>
+        <span translate *ngIf="!record.metadata.available">not available</span>
+      </div>
+      <div *ngIf="record.explanation">
+        <a class="badge badge-info collapsed"
+            data-toggle="collapse" href="#{{'score'+record.metadata.pid }}"
+            aria-expanded="false">
+            score: {{ record.explanation.value }}
+        </a>
+        <pre class="collapse border border-secondary mt-1" id="{{'score'+record.metadata.pid }}">{{record.explanation|json}}</pre>
```

Comment:
Missing space  after `{{` on first element and second missing space before ` }}` and after `{{`